src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.ps1

```diff
@@ -226,6 +226,43 @@ function Test-AzureRmIotHubLifecycle
 	$iotHubUpdated = Set-AzureRmIotHub -ResourceGroupName $ResourceGroupName -Name $IotHubName -FallbackRoute $iothub.Properties.Routing.FallbackRoute	
     Assert-True { $iotHubUpdated.Properties.Routing.FallbackRoute.IsEnabled -eq 1}
 
+	# Add Certificate
```

Comment:
Please make this into separate tests - if any of these cmdlets cause the test to fail it will be difficult to figure out what is making the test fail.

Reply:
Separated in two tests